rust/scx_rustland_core/assets/bpf/main.bpf.c

```diff
@@ -432,8 +431,8 @@ dispatch_task(struct task_struct *p, u64 dsq_id,
 		dbg_msg("dispatch: pid=%d (%s) dsq=%llu enq_flags=%llx slice=%llu",
 			p->pid, p->comm, dsq_id, enq_flags, slice);
 
-		/* Wake up the target CPU (only if idle) */
-		scx_bpf_kick_cpu(cpu, __COMPAT_SCX_KICK_IDLE);
+		/* Wake up the target CPU */
+		scx_bpf_kick_cpu(cpu, 0);
```

Comment:
We should root cause why this makes a difference as this is indicating a bug either in SCX core or rustland.

Reply:
I'll investigate more, I also want to understand better what's going on, so let's wait a bit before merging this.